config/config.go

```diff
@@ -0,0 +1,81 @@
+package config
+
+import (
+	"os"
+	"path"
+	"strings"
+
+	"github.com/daedaleanai/dbt/log"
+	"github.com/daedaleanai/dbt/util"
+	"gopkg.in/yaml.v2"
+)
+
+type Config struct {
+	Mirror string
+}
+
+var environment map[string]string
+var config *Config
+
+const configFileName string = "config.yaml"
+
+func init() {
+	environment = make(map[string]string)
+	for _, v := range os.Environ() {
+		parts := strings.Split(v, "=")
+		if len(parts) == 2 {
+			key := parts[0]
+			value := parts[1]
+			environment[key] = value
+		}
+	}
+}
+
+func getDbtConfigDir() string {
+	dbtConfigDir, ok := environment["DBT_CONFIG_DIR"]
+	if ok {
```

Comment:
nit: you can do this in one line if you like: `if dbtConfigDir, ok := environment["DBT_CONFIG_DIR"]; ok {`

Reply:
hmm good point. I'm not sure why I decided to separate it